pkgs/by-name/gu/guymager/package.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, stdenv
+, fetchurl
+, libsForQt5
+, dpkg
+, parted
+, udev
+, libewf-legacy
+, libbfio
```

Comment:
this doesn't exist in the nixpkgs rev you're using, please rebase on current `master`

Reply:
Just for confirm, the right commands to do this are:
```
git clone https://github.com/D3vil0p3r/nixpkgs -b patch-9
git remote add upstream https://github.com/NixOS/nixpkgs.git
git fetch upstream
git rebase upstream/master
git push -f origin patch-9
```
?

Reply:
After `git rebase upstream/master`, there are a lot of files that need to fix conflicts... 
```
...
pkgs/top-level/release-lib.nix: needs merge
pkgs/top-level/release-python.nix: needs merge
pkgs/top-level/release-r.nix: needs merge
pkgs/top-level/release-small.nix: needs merge
pkgs/top-level/release.nix: needs merge
pkgs/top-level/ruby-packages.nix: needs merge
pkgs/top-level/splice.nix: needs merge
pkgs/top-level/stage.nix: needs merge
pkgs/top-level/unixtools.nix: needs merge
pkgs/top-level/wine-packages.nix: needs merge
You must edit all merge conflicts and then
mark them as resolved using git add
```
Probably it is better to close this PR and open a new one?

Reply:
GitHub interface is sub subpar.

You can use the bulldozer method:

- Save guymager/package.nix to a safe location (say ~/backups/)
- create a local `patch-9` branch
- get back guymager/package.nix
- commit and force-push

Reply:
@D3vil0p3r you should already have a local clone of nixpkgs if you're creating PRs, as rebasing is often necessary.
Ensure you _do not_ use a shallow or treeless clone; use a blobless clone if you need to save time (`--filter=blob:none`)

Reply:
So, if I understood well, I should blobless clone the nixpkgs repo of my `patch-9`:
```
git clone https://github.com/D3vil0p3r/nixpkgs -b patch-9 --filter=blob:none
cd nixpkgs
git rebase -i
```
but I get only `noop`. Should I do additional things? Sorry for wasting your time guys.